website/docs/r/advanced_cluster.html.markdown

```diff
@@ -181,7 +181,7 @@ resource "mongodbatlas_advanced_cluster" "cluster" {
         node_count    = 1
       }
       provider_name = "GCP"
-      priority      = 0
+      priority      = 5
```

Comment:
just curious, why are we changing advanced cluster priority in this doc example?